modules/tensor_mechanics/tests/finite_strain_elastic/elastic_rotation_test.i

```diff
@@ -21,55 +26,50 @@
   [./x_200]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, delta*t, (1.0+delta)*cos(pi/2*(t-t0)) - 1.0)'
   [../]
   [./y_200]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, 0.0, (1.0+delta)*sin(pi/2*(t-t0)))'
   [../]
   [./x_300]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, delta*t, (1.0+delta)*cos(pi/2.0*(t-t0)) - sin(pi/2.0*(t-t0)) - 1.0)'
   [../]
   [./y_300]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, 0.0, cos(pi/2.0*(t-t0)) + (1+delta)*sin(pi/2.0*(t-t0)) - 1.0)'
   [../]
   [./x_400]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, 0.0, -sin(pi/2.0*(t-t0)))'
   [../]
   [./y_400]
     type = ParsedFunction
     vars = 'delta t0'
-    vals = '-1e-6 1.0'
+    vals = '1e-6 1.0'
     value = 'if(t<=1.0, 0.0, cos(pi/2.0*(t-t0)) - 1.0)'
   [../]
 []
 
-[Modules]
-  [./TensorMechanics]
-    [./Master]
-      [./all]
-        strain = FINITE
-        add_variables = true
-        #generate_output = 'stress_xx stress_yy stress_zz stress_xy stress_yz stress_zx'
-      [../]
-    [../]
```

Comment:
I like this change, but currently the python based input file parser that Peacock and the documentation system uses does not understand this type of syntax. We are working on changing this, but it will take a bit of time. I suggest leaving the current syntax as is for now. I will add a comment to update this syntax to #8365.